Docs/source/Software/BSP.rst

```diff
@@ -135,3 +135,18 @@ Usage
 Additional Considerations
     Increasing the clock speed does take more power. Although this is minor considering how much the
     entire BPS takes, it is worth noting.
+
+SPI BSP: Clark Poon, Sijin Woo, and Sugam Arora
+===============================================
+
+Purpose
+    SPI is used for communication between the uC and the minion LTC6811s in the system.
+
+Usage
+    Our car uses the ``SPI1`` and ``SPI3`` busses. When calling the init function, you must specify which 
+    port you intend to use (via the ``spi_port_t`` struct). The user can read/write on whichever bus they 
+    have chosen. 
+    This BSP includes IRQ Handlers for both buses that post the ``MinionsASIC_Mutex``.
+
+Additional Considerations
+    The ``SPI1`` and ``SPI3`` IRQs are only included in the RTOS version of the code.
```

Comment:
If there is any reason they should be added to the Bare-Metal version of the code I would include that.

Reply:
Last spring the team decided to leave these handlers out of the bare metal version